docs/core/deploying/acquisition-experiences.md

```diff
@@ -0,0 +1,179 @@
+
+Differences between the various choices covered above make some of them a great option for developers while others are better suited for DevOps engineers and IT administrators. 
+
+
+|Acquisition Choice|Great For Developers|Great For DevOps/ IT Admins|
+|:-|:-:|:-:|
+[Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)   | ✔️ | ❌ |
+[https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)        | ✔️ | ✔️ |
+[Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)                | ✔️ | ✔️ |
+[Docker](#Docker)                                                               | ✔️ | ✔️ |
+[Linux Package Managers](#Linux-Package-Managers)                               | ✔️ | ✔️ |
+[Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)   | ❌ | ✔️ |
+[Snap store](#Snap-store)                                                       | ✔️ | ✔️ |
+[.NET Core on GitHub](#NET-Core-on-GitHub)                                      | ✔️ | ❌ |
+
+
+Let's look at each of these choices for .NET Core acquisition in further detail.
+
+## Visual Studio and Visual Studio Code
+
+Developers often acquire and update the .NET Core tools and runtime by installing Visual Studio or Visual Studio for Mac. 
+
+### Visual Studio for Windows
+Each version of Visual Studio installs the latest .NET Core SDK tools, and one or more runtimes so you can target your app to run on the .NET Core runtime version of your choice.
+
+Selecting any of the managed workloads for Web, Desktop or Azure development will install the .NET Core development tools and runtime.
+
+![Install Visual Studio for Windows](media/install-vs-windows.png "Install Visual Studio for Windows")
+
+In recent previews of Visual Studio 2019 Update 3, we have made changes to the installation experience for .NET Core in order to make the experience more streamlined. For more details, see the blog post [Improving .NET Core installation in Visual Studio and on Windows](https://devblogs.microsoft.com/dotnet/improving-net-core-installation-in-visual-studio-and-on-windows/). 
+
+
+
+### Visual Studio for MacOS
+
+The latest release version of the .NET Core tools are installed when you install Visual Studio for Mac and select the .NET Core workload. To get started with .NET Core development on MacOS, see [Install Visual Studio 2019 for Mac](https://docs.microsoft.com/visualstudio/mac/installation?view=vsmac-2019).
+
+![Install Visual Studio for Mac](media/install-vs-mac.png "Install Visual Studio for Mac")
+
+Note: .NET Core 2.x development is supported on MacOS 10.12 “Sierra” and later. 
+
+For more information about .NET Core development on MacOS, see [Get started with .NET Core on macOS using Visual Studio for Mac](https://docs.microsoft.com/dotnet/core/tutorials/using-on-mac-vs).
+
+
+## Visual Studio Code
```

Comment:
Does this really belong? Yes, it's a developer install experience, but since .NET is not installed as part of VS Code itself, wouldn't this just fall under the install from dot.net umbrella?

Reply:
Didn't have this previously, but added by popular demand since VS code is believed to be a popular choice for Code development. We do expect the dev to install the SDK and the C# extension for VSCode.